.github/pull_request_template.md

```diff
@@ -10,4 +10,3 @@ Given this is a project maintained by volunteers, please read this template to n
 - [ ] I understand that this PR may be closed in case there was no previous discussion. (This doesn't apply to typos!)
 - [ ] I've added a test for each change that was introduced, and I tried as much as possible to make a single atomic change.
 - [ ] I've updated the documentation accordingly.
-- [ ] I've added a note on the `docs/release-notes.md` about my changes (changes on documentation or type annotations don't need this).
```

Comment:
I'm removing this here since I noticed this will be a problem on future PRs, since we have a script that checks the version on `__init__.py` against the first version found on the release notes.